usr/bin/nxdk-configure

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/sh
+
+#FIXME: add explicit support for SDL / SDL2?
+#       https://wiki.libsdl.org/FAQLinux#how_do_i_add_sdl_to_my_project
+
+export CC=nxdk-cc
+export PKG_CONFIG=nxdk-pkg-config
+
+# Wrapper around autoconf ./configure scripts
+./configure --host=i686-windows
```

Comment:
This uses host `ar` and builds .a instead of .lib.
A bit strange, but not a problem.

When using i686-mingw32 it will attempt to build a .lib and .dll (but it fails and I didn't check why that is).